chat-plugins/info.js

```diff
@@ -1682,11 +1682,25 @@ const commands = {
 	rule: 'rules',
 	rules: function (target, room, user) {
 		if (!target) {
+			const languageTable = {
```

Comment:
I disagree, because these commands only support some languages I don't really see why making it a config setting makes sense. I consider Config a file where you an object you can edit to customize your server, if you wanted to use more languages it would be more work than adding them to config.js

Reply:
Yeah for the scope of this PR this implementation is 100% fine. It's rather pointless to discuss "somewhere down the road" now because it's nothing we can't easily adjust for that anyway.